lib/cog_api/http/rules.ex

```diff
@@ -17,6 +17,16 @@ defmodule CogApi.HTTP.Rules do
     |> ApiResponse.format(%Rule{})
   end
 
+  def show(rule_id, %Endpoint{}=endpoint) do
+    Base.get(endpoint, "rules/#{rule_id}")
+    |> ApiResponse.format(%Rule{})
+  end
+
+  def update(rule_id, rule_text, %Endpoint{}=endpoint) do
+    Base.patch(endpoint, "rules/#{rule_id}", %{rule: rule_text})
```

Comment:
Yep, makes it nice and atomic without having overlapping or missing rules in the middle of an update.